spec/valid_email2_spec.rb

```diff
@@ -115,4 +115,17 @@ class BackwardsCompatibleUser < TestModel
       expect(user.valid?).to be_falsey
     end
   end
+
+  describe "emoticons emails" do
+    it "should be invalid if email contains emoticon" do
+      email = ValidEmail2::Address.new("foo🙈@gmail.com")
+      expect(email.valid?).to be_falsy
+      end
+
+    it "should be invalid if email contains parsed emoticon" do
```

Comment:
This test is not really testing any parsing of an emoticon.
The email is actually invalid only because it includes a colon `:`